### PR TITLE
Added command to add new client to existing API scope

### PR DESCRIPTION
**Pull request by Hi-Fi**
Needed to make Tunnistamo available for review environments

### CODE REVIEW

File: utils/management/commands/add_oidc_client_to_api_scope.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            "-asi",
+            "--api_scope_identifier",
+            type=str,
+            help="API Scope Identifier",
+            required=True,
+        )
+
+    @transaction.atomic
+    def handle(self, *args, **kwargs):
+        apiScope = ApiScope.objects.filter(identifier=kwargs["api_scope_identifier"])
```
Comment (tuomas777) [inline utils/management/commands/add_oidc_client_to_api_scope.py:25]: It is better to get the first object here already, ie.
`api_scope = ApiScope.objects.filter(identifier=kwargs["api_scope_identifier"]).first()`

File: utils/management/commands/add_oidc_client_to_api_scope.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        if not apiScope:
+            print("No API Scope with that identifier found")
+            return False
+
+        clients = Client.objects.filter(client_id=kwargs["client_id"])
```
Comment (tuomas777) [inline utils/management/commands/add_oidc_client_to_api_scope.py:30]: Ditto.

File: utils/management/commands/add_oidc_client_to_api_scope.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        )
+        parser.add_argument(
+            "-asi",
+            "--api_scope_identifier",
+            type=str,
+            help="API Scope Identifier",
+            required=True,
+        )
+
+    @transaction.atomic
```
Comment (tuomas777) [inline utils/management/commands/add_oidc_client_to_api_scope.py:23]: This seems unnecessary.

File: utils/management/commands/add_oidc_client_to_api_scope.py (hunk 1 above)
Comment (Hi-Fi) [reply utils/management/commands/add_oidc_client_to_api_scope.py]: Getting first at here.

File: utils/management/commands/add_oidc_client_to_api_scope.py (hunk 3 above)
Comment (Hi-Fi) [reply utils/management/commands/add_oidc_client_to_api_scope.py]: Removed. Originates from other oidc-client related actions.

File: utils/management/commands/add_oidc_client_to_api_scope.py (hunk 2 above)
Comment (Hi-Fi) [reply utils/management/commands/add_oidc_client_to_api_scope.py]: Getting first at here.

**Review by tituomin — APPROVED** [review]

**Review by tuomas777 — APPROVED** [review]
LGTM 👍

**Pull request merged**